packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -0,0 +1,384 @@
+  </CalloutText>
+  <CalloutText>
+    Anecdotal data have also suggested that some Twilio customers find it annoying to have to manually close success
+    toasts.
+  </CalloutText>
+</Callout>
+
+### Warning Toast
+
+Use a warning toast to help a user avoid a potential future issue that might happen based on their action. If the user can take an immediate action to avoid the issue, provide a way to do that. In most cases, a warning toast lets users know that their current action completed but may have triggered a related issue, or that their action can't be completed because of an issue that they need to fix outside their current flow.
+
+Be cautious when using a warning toast because they can be stressful for a user to see. If the information you're trying to convey is not very critical, consider using a neutral toast instead.
+
+<LivePreview scope={{Toast, Text, Anchor}} language="jsx">
+  {`<Toast onDismiss={() => alert('dismissed')} variant="warning">
+  <Text as="span">
+    The phone number has been deleted. Please verify there are no functions tied to that number <Anchor href="#">here</Anchor>.
+  </Text>
+</Toast>`}
+</LivePreview>
+
+### Error Toast
+
+Use an error toast to help a user understand an issue that's preventing them from completing their current action. Provide a way to resolve the issue either in the underlying page content or in the toast.
+
+Be cautious when using an error alert because they can be stressful for a user to see.
+
+<LivePreview scope={{Toast, Text}} language="jsx">
+  {`<Toast onDismiss={() => alert('dismissed')} variant="error">
+  <Text as="span">
+    There was an error when deleting the profile. Please try again.
+  </Text>
+</Toast>`}
+</LivePreview>
+
+### Toast with automatic time-out
+
+Add an automatic time-out to a toast if it doesn't communicate any information that would help a user continue a flow, and would be annoying to the user to have to explicitly acknowledge and close.
+
+In most cases, it's better to dismiss the toast when a user reloads or navigates away from a page, rather than timing it out. Add an automatic time-out to a success toasts if the only thing it does it confirm that the action was completed. If the toast has any additional information or interactive content, like a link or button, don't make the toast time out.
+
+Add an automatic time-out if all these conditions are true:
+
+- The user wouldn't need to remember or copy the information in order to continue with their current flow, or any future flow.
+- The toast doesn't have any interactive content, such as a link or button.
+
+We recommend giving users about 0.5 seconds per word before timing out a toast.
+
+### Showing and Hiding a Toast
+
+For a basic method of hiding and showing, you can use the `hidden` HTML attribute. Passing a true or false value to that attribute will show or hide the Toast respectively.
+We'll be providing a more comprehensive method for displaying Toasts in the near future.
```

Comment:
I'd be tempted to make this a callout at the start of this section to make it _really_ clear that we acknowledge this is just an interim solution.

Reply:
I think thats a good call. Done.